app/assets/stylesheets/layout_components/features.scss

```diff
@@ -27,77 +27,76 @@
     max-width: 100%;
     background-position: center;
 
-    @include breakpoint(small only){
+    @include breakpoint(small only) {
       max-width: 90%;
     }
   }
 
-  &__content{
-    @include breakpoint(large){
+  &__content {
+    @include breakpoint(large) {
       width: 80%;
       margin: 0 auto;
     }
   }
 }
 
-#feature__image{
-  &--devices{
+#feature__image {
+  &--devices {
     background-image: asset-url(gatekeeper/devices);
   }
 
-  &--membership{
+  &--membership {
     background-image: asset-url(gatekeeper/membership);
   }
 
-  &--vrar{
+  &--vrar {
     background-image: asset-url(gatekeeper/vrar);
   }
 
-  &--automotive{
+  &--automotive {
     background-image: asset-url(gatekeeper/automotive);
   }
 
-  &--rewards{
+  &--rewards {
     background-image: asset-url(gatekeeper/rewards);
   }
 
-  &--subscriptions{
+  &--subscriptions {
     background-image: asset-url(gatekeeper/subscriptions);
   }
 
-  &--gift-cards{
+  &--gift-cards {
     background-image: asset-url(gatekeeper/gift-card);
   }
 }
 
-@media (-webkit-min-device-pixel-ratio: 2),
-(min-resolution: 192dpi){
-  #feature__image{
-    &--devices{
+@media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
```

Comment:
So is the idea here that multiple selectors / classes which are separated by commas should be on new lines, but media query attributes separated by commas are on the same line?

Reply:
yeah, my thought was that here they both relate back to the @media so it made sense, whereas usually if we have different classes separated by commas they aren't (necessarily) related to each other